src/modules/10-common/components/ResourceCardList/__tests__/ResourceCardList.test.tsx

```diff
@@ -83,4 +83,36 @@ describe('Resource card list test', () => {
     expect(getByText('common.repositoryCertificates')).toBeDefined()
     expect(getByText('common.gnupgKeys')).toBeDefined()
   })
+
+  test('it should not render the Feature Flags Proxy tile at account level when the flag is false', () => {
+    const renderComponent = (): RenderResult =>
+      render(
+        <TestWrapper
+          path={ACCOUNT_LEVEL_GITOPS_ROUTE}
+          pathParams={ACCOUNT_PATH_PARAMS}
+          defaultFeatureFlagValues={{ FFM_9497_PROXY_KEY_MANAGEMENT: false }}
+        >
+          <ResourceCardList />
+        </TestWrapper>
+      )
+
+    renderComponent()
```

Comment:
Extracting the `renderComponent` function on a per-test basis isn't really needed. I'd just render directly.
```suggestion
    render(
      <TestWrapper
        path={ACCOUNT_LEVEL_GITOPS_ROUTE}
        pathParams={ACCOUNT_PATH_PARAMS}
        defaultFeatureFlagValues={{ FFM_9497_PROXY_KEY_MANAGEMENT: false }}
      >
        <ResourceCardList />
      </TestWrapper>
    )
```

Reply:
Thank you, have addressed